calico-3.28.yaml

```diff
@@ -64,13 +64,7 @@ pipeline:
     with:
       repository: https://github.com/projectcalico/calico
       tag: v${{package.version}}
-      expected-commit: 413e6f5593a8e76d34cf5338a642265da52561ce
-  - uses: go/bump
-    with:
-      deps: k8s.io/kubernetes@v1.28.12
-  - uses: patch
```

Comment:
Patch removed, as our upstream contribution is now in the latest release
https://github.com/projectcalico/calico/pull/8693